components/x-gift-article/src/ShareArticleDialog.scss

```diff
@@ -179,3 +179,39 @@
 		@include oTypographySans($weight: 'medium');
 	}
 }
+
+.share-article-modal__close {
```

Comment:
I think we should leave a short comment in the css here explaining that these styles have been copied directly from o-overlay close as we're no longer using o-overlay but want to preserve close functionality and a11y

Reply:
Here is the PR for the comment https://github.com/Financial-Times/x-dash/pull/746